modules/RTC/ScreenObtainer.js

```diff
@@ -214,9 +214,10 @@ const ScreenObtainer = {
             getDisplayMedia = navigator.mediaDevices.getDisplayMedia.bind(navigator.mediaDevices);
         }
 
-        const { desktopSharingFrameRate } = this.options;
+        const { desktopSharingFrameRate, desktopSharingAudio } = this.options;
         const video = typeof desktopSharingFrameRate === 'object' ? { frameRate: desktopSharingFrameRate } : true;
-        const audio = this._getAudioConstraints();
+        // eslint-disable-next-line no-negated-condition
+        const audio = typeof desktopSharingAudio !== 'undefined' ? desktopSharingAudio : this._getAudioConstraints();
```

Comment:
Why are adding a new `desktopSharingAudio` option ? Doesn't this affect audio-only screenshare feature ? Are making sure this option is set to true for that feature ?